lib/web/cortina.js

```diff
@@ -24,10 +24,14 @@ module.exports = function cortinaCommon(options) {
     proxyPrefixPath,
     hostUrl,
     redisConfig,
-    redisKey,
     siteNameKey = 'site_name',
     localeTextKey = 'locale_text',
+    styleVersion = 9,
   } = options
+
+  const blockView = styleVersion == 10 ? 'style10' : 'style9'
```

Comment:
I’m assuming the type coercion is by design. Is it to clever though? Easily missed by a tired developer on a Friday afternoon?